lib/aws/active_job/sqs/configuration.rb

```diff
@@ -3,11 +3,57 @@
 module Aws
   module ActiveJob
     module SQS
-      # Use +Aws::ActiveJob::SQS.config+ to access the singleton config instance.
+      # This class provides a Configuration object for AWS ActiveJob
+      # by pulling configuration options from runtime code, the ENV, a YAML file,
+      # and default settings, in that order. Values set on queues are used
+      # preferentially to global values.
+      #
+      # Use {Aws::ActiveJob::SQS.config Aws::ActiveJob::SQS.config}
```

Comment:
Redundant. I think just the class and then # method? Other instances below

Reply:
It needs to be the `.` instead of `#` because its a class method and not an instance method.

I also don't think its redundant.  Without the name the link title becomes jsut "config" instead of the full name which they would actually need to use in code.